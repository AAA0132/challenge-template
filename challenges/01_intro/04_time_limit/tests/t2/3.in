877745430909101787055
