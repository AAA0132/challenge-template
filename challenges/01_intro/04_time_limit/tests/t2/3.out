385218520740902389406031310978751833680040
